src/Analyzers/Core/CodeFixes/UseCompoundAssignment/AbstractUseCompoundAssignmentCodeFixProvider.cs

```diff
@@ -43,6 +43,7 @@ internal abstract class AbstractUseCompoundAssignmentCodeFixProvider<
             TSyntaxKind assignmentOpKind, TExpressionSyntax left, SyntaxToken syntaxToken, TExpressionSyntax right);
         protected abstract TExpressionSyntax Increment(TExpressionSyntax left, bool postfix);
         protected abstract TExpressionSyntax Decrement(TExpressionSyntax left, bool postfix);
+        protected abstract SyntaxTriviaList PrepareRightExpressionLeadingTrivia(SyntaxTriviaList initialTrivia);
```

Comment:
nit: maybe just call this `RemoveLeadingWhitespaces` for clarity.

Reply:
Not a good idea, considering the fact, that this method now also trims leading new lines for C#. More generic name is preffered here IMO